packages/react-dom/src/server/ReactDOMFizzServerNode.js

```diff
@@ -36,7 +37,14 @@ function createAbortHandler(request: Request, reason: string) {
   return () => abort(request, new Error(reason));
 }
 
-type Options = {
```

Comment:
I usually try to stick to `${ucfirst(methodName)}Options` for naming. Gets a bit more tricky when we have shared options but overall I like that the verbose name requires less context (see "? Options" or "? toReadableOptions")

Reply:
Since we're export flow types and not TS types does a name change here actually ever break anything? I guess for flow users it could.

Should I just export the options now following the existing convention?

```
export type RenderToReadableStreamOptions = ...
export type RenderIntoContainerOptions = ...
```

Reply:
It seems like the Options convention in JS is just a hack around named arguments anyway and it doesn't actually have its own nominal structure. So it makes sense that each function gets their own. As we've seen, they can also change independently.

That convention shouldn't necessarily carry over to any other type than "Options" though.